Makefile

```diff
@@ -7,6 +7,18 @@ cppflags := -DDATADIR=$(DATADIR)
 
 sources := $(wildcard src/brogue/*.c) $(addprefix src/platform/,main.c platformdependent.c)
 
+ifeq ($(shell git tag --points-at HEAD),)
+ifeq ($(CI),)
+	commit := dirty
+	gitref := $(shell git rev-parse --abbrev-ref HEAD)
+else
+	commit := $(shell echo $(GITHUB_SHA) | cut -c 1-7)
+	gitref := $(shell echo $(GITHUB_REF) | cut -c 12-)
```

Comment:
What's the reason to split this by CI / not CI?

Reply:
I wanted to mark local builds (i.e. me running `make -j`) in some way, to help me remember when I'm not playing vanilla.  

Since my git work tree might be dirty (`git status` is not empty), sticking in the local commit wouldn't be exactly appropriate (going back to the tester use case.)

Likewise it might serve as weak protection against a script kiddie tweaking the game in their favor, though a real version of that would involve some kind of code signing.

I'd be content marking local builds simply `dirty` (or `local` if you like, or perhaps something else) and not shell out to git.

Reply:
Opting to mark them `unstable`, in line with the other uses of that word.